<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>custom</class>
 <widget class="QDialog" name="custom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>361</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>运行周期:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_zdy">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_zdy_ks">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zdy_yxzq">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>361</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
